Sources/Client/Util/Updater.swift

```diff
@@ -266,6 +266,39 @@ public final class Updater: ObservableObject {
       }
     }
   }
+  
+  /// Check if a commit (by its SHA) exists on a given branch.
+  ///
```

Comment:
This middle line of the documentation comment is unnecessary

Reply:
Sure, I just saw that you had a blank middle line in some of your other comments.

Reply:
Yeah, those are from before I realised and I haven't fixed them yet 😅. Thanks for following the code style of the codebase so well though, not everyone is observant enough to do that and it leads to PRs taking longer to accept.